developer-docs-site/docs/reference/glossary.md

```diff
@@ -236,6 +236,12 @@ then there is a guarantee that T_N will never be included in the blockchain.
 
 # R
 
+## Resource Account
+
+- A resource account is used to manage resources independent of an account managed by a user. For example, a developer may use a resource account to manage an account for module publishing, say managing a contract. The contract itself does not require signer post initialization.
```

Comment:
`does not require a signer`

It's a little weird, but let's keep it (and add the article `a`)

Reply:
Done! Thanks for the review.